lib/common/utils.js

```diff
@@ -156,6 +156,16 @@ export const relativeTo = function relativeTo() {
   // Resolve path
   let _path = r(...args)
 
+  // Keep webpack inline loader intact
+  if (_path.indexOf('!') !== -1) {
+    const [loader, __path, ...rest] = path.split('!')
+    if (rest.length) {
+      throw new Error('Unexpected use of multiple "!" in path')
```

Comment:
@Xiphe Thanks for your contribution.
I have a question: if use multiple inline loader such like `vue-loader!custom-loader?modules!./test.vue`, here may throw exception, so do you thinks we need to consider multiple loaders introduced by multiple modules?